externs/ngeox.js

```diff
@@ -190,6 +214,54 @@ ngeox.profile.PoiExtractor.prototype.sort;
 ngeox.profile.PoiExtractor.prototype.title;
 
 
+
+/**
+ * @typedef {{
+ *   xhover: function(number, string): string,
+ *   yhover: function(number, string): string,
+ *   xtick: function(number, string): (string|number),
+ *   ytick: function(number, string): (string|number)
+ * }}
+ */
+ngeox.profile.ProfileFormatter;
+
+
+/**
+ * Format the xhover distance
+ * @param {number} xdistance
+ * @param {string} units
+ * @return {string}
```

Comment:
There are two issues here: using a `@type` annotation and allowing `undefined` functions.
I updated the commit to use `@type`.
However not allowing `undefined` is on-purpose: if the application developer does not want to use the defaults, then I expect all the functions of the ProfileFormatter to be defined.
Do you think I should allow a finer granularity?